### PR TITLE
add step to publish to static when...

### PR DESCRIPTION
**Pull request by nickboldt**
add step to publish to static when publishing to Marketplace; remove unneeded code (?) too

Change-Id: Iae731a81524bf8e416c0af67170b7378977efd47
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -59,11 +58,18 @@ node('rhel7'){
...
+		unstash 'vsix'
 		withCredentials([[$class: 'StringBinding', credentialsId: 'vscode_java_marketplace', variable: 'TOKEN']]) {
 			def vsix = findFiles(glob: '**.vsix')
 			sh 'vsce publish -p ${TOKEN} --packagePath' + " ${vsix[0].path}"
 		}
 		archive includes:"**.vsix"
-	}//if publishMarketPlace
+	}// if publishToMarketPlace
+
+	if(publishToStable.equals('true')){
```
Comment (fbricon) [inline Jenkinsfile]: We don't need a new publishToStable  parameter. This block should be executed if package was successufully pushed to the marketplace

Comment (nickboldt) [reply Jenkinsfile]: yes, that was just for my own testing. Here's the cleaned up / squashed PR. https://github.com/redhat-developer/vscode-java/pull/552/commits/a1da5e3da8a2b349b3231e38baca3a93936c00ac

**Comment by nickboldt** [comment]
Note that UPLOAD_LOCATION has already been added as a param on https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Adapters/view/VSCode/job/vscode-java_release/configure so once merged this should Just Work (tm).

**Comment by fbricon** [comment]
Thanks for the patch @nickboldt!

**Pull request merged**